src/Mapper/SyliusSectionsToPermissionsMapper.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\RbacPlugin\Mapper;
+
+use Sylius\RbacPlugin\Access\Model\Section;
+use Sylius\RbacPlugin\Model\Permission;
+
+final class SyliusSectionsToPermissionsMapper implements SectionsToPermissionsMapperInterface
+{
+    /** @var array */
+    private $sectionsToPermissionsMapping;
+
+    public function __construct()
+    {
+        $this->sectionsToPermissionsMapping = [
```

Comment:
It could be initialized not in the constructor, but in `private $sectionsToPermissionsMapping;` 🚀 It could then also be a static variable, I suppose